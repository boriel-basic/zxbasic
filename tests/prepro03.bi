
#define test x 'dasfdfasfdsa \
#define test2 y // This should be ignored

test
test2




