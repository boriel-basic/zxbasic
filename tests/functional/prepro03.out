#line 1 "prepro03.bi"




x 
test2




